web/templates/settings/bot_settings_tip.hbs

```diff
@@ -1,7 +1,19 @@
 {{#unless can_create_any_bots}}
     {{#if can_create_incoming_webhooks}}
-        <div class='tip'>{{t "You can create bots that can only send messages." }}</div>
+        {{> ../components/banner
```

Comment:
The banners should always be wrapped inside a `banner-wrapper` parent for the container queries to work.

Reply:
FYI, You can add the `banner-wrapper` class to the pre-existing parent, and don't always have to introduce a new wrapper div.

Reply:
@shuklamaneesh23 have you resolved this feedback?